plugins/generator-1.16.5/forge-1.16.5/mappings/types.yaml

```diff
@@ -1,14 +1,14 @@
-_default: Object
-number: double
-string: String
-logic: boolean
-blockstate: BlockState
-itemstack: ItemStack
-direction: Direction
-world: IWorld
-entity: Entity
-map: HashMap
-advancement: Advancement
-dimensiontype: RegistryKey<World>
-actionresulttype: ActionResultType
+_default: Object
```

Comment:
What did change in this file? If nothing, I suggest reverting to head files like this one

Reply:
Specifically: change line separator <kbd>LF</kbd> back to <kbd>CRLF</kbd>.

Reply:
Or `git checkout <head hash> <file>`

Reply:
> What did change in this file? If nothing, I suggest reverting to head files like this one

It was due to what Defeatomizer said.